### DISCUSSION TITLE
Select color for button background on Horizontal button stack

### DISCUSSION THREAD

**Discussion by sirmeili** [Clooos/Bubble-Card#295] [Feature requests]
I'm trying out the card, which is great, BTW.

one thing I noticed is that when I chose a normal light (dimmer, but no white balance or color control), it appears to use a gray for th background of the button. Is there a way to make this configurable or have it at least default to yellow? Or am I missing something?

![image](https://github.com/Clooos/Bubble-Card/assets/6846886/5c9de964-c0e7-466a-a8ac-8094dbf200e3)

**Comment by tonka3000** [comment]
I would also love to adjust the color and maybe add some state.
